modules/rosapps/lib/vfdlib/vfdguisave.c

```diff
@@ -293,7 +297,11 @@ void OnTarget(
 	//
 	//	get the current image info
 	//
+#ifndef __REACTOS__
+	param = (PCSAVE_PARAM)GetWindowLong(hDlg, GWL_USERDATA);
```

Comment:
same: better propose our fix upstream instead of marking the diff?

Reply:
Same: see my reply to your previous comment.